demo/demo.ts

```diff
@@ -1,4 +1,53 @@
 // THIS IS DEMO FILE ONLY FOR DEV TESTING
-import { name } from '../src';
+import { Coordinates3D } from '../src/services/HandTrackingService/HandTrackingService.type';
+import { HandTrackingService } from '../src/services/HandTrackingService/HandTrackingService';
 
-console.log(name);
+const HTS = new HandTrackingService();
+
+const intervalDelay = 30;
+
+const initPoints = (pointsNumber: number) => {
+  const pointsArray = [];
+  const parent = document.querySelector('#points');
+  for (let i = 0; i < pointsNumber; i++) {
+    const newPoint = document.createElement('div');
+    pointsArray.push(newPoint);
+    newPoint.classList.add('point');
+    const whichPointsAreTips = 4;
+    if (!(i % whichPointsAreTips)) {
+      newPoint.style.backgroundColor = 'green';
+    }
+    parent?.appendChild(newPoint);
+  }
+
+  return pointsArray;
+};
+
+const movePoints = (points: HTMLElement[], positions: Coordinates3D[][]) => {
+  const whichHand = 0;
+  points.forEach((element: HTMLElement, id: number) => {
+    element.style.left = `${positions[whichHand][id]?.x * screen.width}px`;
+    element.style.top = `${positions[whichHand][id]?.y * screen.height}px`;
+  });
+};
+
+const loader = async () => {
+  await HTS.start();
+  const numberOdPointsInhand = 21;
```

Comment:
a typo, should be `numberOfPointsInHand`

Reply:
thx, fixed